mappings/net/minecraft/structure/BastionBridgeData.mapping

```diff
@@ -0,0 +1,2 @@
+CLASS net/minecraft/class_5174 net/minecraft/structure/BastionBridgeData
+	METHOD method_27198 init ()V
```

Comment:
`initialize`, to match the village data classes

Reply:
See #618, where we've decided to use `init` globally.